tests/test_file.py

```diff
@@ -624,3 +633,62 @@ def test_file_zarr():
     file.save()
     file.delete(storage=False)
     UPath("test.zarr").unlink()
+
+
+def test_zarr_folder_upload():
```

Comment:
Is this an actual folder, or is it a zarr object (represented as a folder)?

Could you add a comment here?

Reply:
It is a zarr object represented as an actual folder.